esp/services/esdl_svc_engine/esdl_store.cpp

```diff
@@ -838,9 +869,13 @@ class CEsdlSDSStore : implements IEsdlStore, public CInterface
         }
         else //espPort provided
         {
-            if (bindingcfg)
+            const char *bn = nullptr;
+            if (isContainerized())
+                bn = "esdl_svc_engine_binding";
```

Comment:
Why limit to the fixed binding esdl_svc_engine_binding for isContainerized? Shouldn't all the bindings it loads from dali be overwrite-able?

Reply:
In k8s ESP is currently limited to one port per pod.  It works better to allow each port to scale independently.  If we eventually we want to enable more than one port per pod we can.  I don't really see the need given how easy it is to startup new pods and services and how we want to scale each port based on need.

Reply:
OK then it makes sense.